Neos.Neos/Migrations/Mysql/Version20170115114801.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Rename Typo3BackendProvider to Neos.Neos:Default
+ */
+class Version20170115114801 extends AbstractMigration
```

Comment:
Doctrine migrations should have a `getDescription` method (`./flow doctrine:migrationgenerate --diff-against-current false` will generate it)